daisy_workflows/export/export_sbom.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# SBOM export work is done in this script, regardless of linux distro
+
+usage() {
+  echo "Usage: export_sbom.sh --syft-source path_to_syft_source --sbom-path path_to_export_sbom"
+  echo "ExportFailed: incorrect SBOM script arguments"
+  exit 1
+}
+
+if [ $# -ne 4 ]; then
```

Comment:
I'd suggest start using `getopt` for parameter parsing (specially when using flags like here) and making things more obvious, I wouldn't block this PR because of that since it's not being used across the code base though.

Reply:
changed to getopt